app/locales/pt-PT.json

```diff
@@ -0,0 +1,188 @@
+  "That way you can update your report if your health condition changes.": "Desta forma poderá atualizar o seu formulário caso o seu estado de saúde se altere.",
+  "I've noted down the link.": "Já guardei o endereço.",
+  "Proceed": "Proceder",
+  "The link has been copied": "O endereço foi copiado.",
+  "Copy": "Copiar",
+  "Could not copy the link, please copy it yourself": "Não nos foi possível copiar o endereço, por favor copie o endereço.",
+  "Contributors": "Voluntários",
+  "This website is made with help from volunteers. In no particular order:": "Este website foi criado com a ajuda de voluntários. Sem nenhuma ordem particular:",
+  "...and others": "...e outros",
+  "The code for the website can be found": "O código do website pode ser encontrado",
+  "here": "aqui",
+  "About the service": "Sobre o serviço",
+  "In connection with the outbreak of COVID-19, many people have been tested for the virus, but due to lack of resources, the overwhelming majority have not been tested. It is therefore assumed that there are large missing numbers here in connection with the outbreak.": "No seguimento do surto de COVID-19 muitas pessoas foram testadas para o vírus, mas devido à falta de recursos a grande maioria não terá sido testada. Supõe-se, portanto, que exista um grande números de casos ausentes das estatísticas atuais sobre o surto.",
+  "Using self-report, you can get a better picture of how many people have symptoms without exposing health workers to potential contamination hazards and without using up valuable infection control equipment that is already in short supply. We at Bustbyte, well helped by other volunteers, have created this tool in response to": "Através da auto avaliação podemos ter uma visão mais abrangente de quantas pessoas têm sintomas sem expor profissionais de saúse a potenciais riscos de contaminação e sem utilizar equipamentos valiosos para controlo da infecção que já estão em falta. Nós na Bustbyte, ajudados por outros voluntários, criámos esta ferramenta em resposta a",
+  "this article": "este artigo",
+  "as an attempt to chart these numbers.": "numa tentativa de mapear estes números.",
+  "This is not an official tool of the authorities, and for more information about the outbreak we refer to health authorities which always have updated information.": "Esta não é uma ferramenta oficial das autoridades e para mais informações sobre o surto deve consultar as autoridades de saúde que têm sempre informações atualizadas, como a",
+  "Developed by": "Desenvolvido por",
+  "BustByte AS": "BustByte AS",
+  "and": "e",
+  "volunteers": "voluntários",
+  "Privacy Policy": "Política de Privacidade",
+  "Privacy": "Privacidade",
+  "Thank you for your contribution!": "Obrigado pelo seu contributo!",
+  "The government has now launched their own self-report system, please use that instead.": "O governo lançou o seu próprio sistema de auto avaliação, por favor utilize-o.",
```

Comment:
I think it's **autoavaliação**, according to [Ciberdúvidas](https://ciberduvidas.iscte-iul.pt/consultorio/perguntas/novo-acordo-autoavaliacao-e-heteroavaliacao/24972)